OTAnalytics/plugin_parser/track_export.py

```diff
@@ -0,0 +1,58 @@
+from pandas import DataFrame
+
+from OTAnalytics.application.use_cases.track_export import (
+    ExportTracks,
+    TrackExportSpecification,
+)
+from OTAnalytics.domain import track
+from OTAnalytics.domain.track_repository import TrackRepository
+from OTAnalytics.plugin_datastore.track_store import (
+    FilteredPandasTrackDataset,
+    PandasTrackDataset,
+)
+
+
+class CsvTrackExport(ExportTracks):
+    def __init__(self, track_repository: TrackRepository) -> None:
+        self._track_repository = track_repository
+
+    def export(self, specification: TrackExportSpecification) -> None:
+        dataframe = self._get_data()
+        dataframe = self._set_column_order(dataframe)
+        output_path = specification.save_path.with_suffix(".tracks.csv")
+        dataframe.to_csv(output_path, index=False)
+
+    def _get_data(self) -> DataFrame:
+        dataset = self._track_repository.get_all()
+        if isinstance(dataset, PandasTrackDataset):
```

Comment:
We could also do an `isinstance` on `PandasDataframeProvider`. The `isinstance` on `FilteredPandasTrackDataset` can then be omitted.

Reply:
It was the wrong import. I move the PandasDataFrameProvider into the pandas module to be more consistent and to prevent circular dependencies.